code/modules/reagents/chemistry/reagents/medicine_reagents.dm

```diff
@@ -503,7 +503,8 @@
 	M.reagents.remove_reagent("nutriment", 5)
 	M.reagents.remove_reagent("vitamin", 5)
 	if(prob(34))
-		M.nutrition -= rand(5, 15)
+		if(M.nutrition > 0)
```

Comment:
Still can technically go into negative 2 > 0 is true and thus you would go to -3 to -12